ReactAndroid/src/main/java/com/facebook/react/views/view/ReactDrawableHelper.java

```diff
@@ -57,6 +53,18 @@ public static Drawable createDrawableFromJSDescription(
     }
   }
 
+  @TargetApi(Build.VERSION_CODES.LOLLIPOP)
+  private static int getAttrId(Context context, String attr) {
+    SoftAssertions.assertNotNull(attr);
+    if ("selectableItemBackground".equals(attr)) {
+      return android.R.attr.selectableItemBackground;
+    } else if ("selectableItemBackgroundBorderless".equals(attr)) {
+      return android.R.attr.selectableItemBackgroundBorderless;
+    } else {
+      return context.getResources().getIdentifier(attr, "attr", "android");
```

Comment:
what about if this return 0?
should we throw an exception as we did in the past?
(A comment from David Vacca @mdvacca.)

Reply:
@mdvacca The output of this is fed into `context.getTheme().resolveAttribute()`on line 42 and if that returns false (which it would for 0), `JSApplicationIllegalArgumentException` is thrown, so it's the same behavior as before.